### DISCUSSION TITLE
有没有对于老毛子padavan的插件格式啊

### DISCUSSION THREAD

**Discussion by Evan-tc** [messense/aliyundrive-webdav#334] [Q&A] [answered]
### 问题描述

有没有对于老毛子padavan的插件格式啊

### 重现步骤

有没有对于老毛子padavan的插件格式啊

### 版本

1

### 运行平台

小米路由r4a

### 日志

有没有对于老毛子padavan的插件格式啊

**Comment by messense** [comment]
无

**Reply by yangjchen** [reply]
大神，求一个padavan的插件格式

**Comment by messense** [comment, marked as answer]
有人做了 Padavan 的集成：https://www.right.com.cn/forum/thread-8227312-1-1.html
